pkg/backup/util/remote.go

```diff
@@ -315,10 +325,45 @@ func BatchDeleteObjectsConcurrently(ctx context.Context, bucket *blob.Bucket, ob
 	return result
 }
 
+func GetStorageCredential(ns string, provider v1alpha1.StorageProvider, secretLister corelisterv1.SecretLister) *storageCredential {
```

Comment:
As it is a public method, I think returning with error or returning an empty object `&storageCredential{}` rather than returning `nil` maybe better. Or it is easy to cause caller encountering nil pointer error.